pkg/cmd/server/kubernetes/master/master_config_test.go

```diff
@@ -47,6 +47,7 @@ var expectedGroupPreferredVersions []string = []string{
 	"rbac.authorization.k8s.io/v1beta1",
 	"settings.k8s.io/v1alpha1",
 	"storage.k8s.io/v1",
+	"user.openshift.io/v1",
```

Comment:
It's based on an imported install.  This package ended up picking it up by accident when the client poked them.